cl/opinion_page/views.py

```diff
@@ -135,17 +183,40 @@ async def court_homepage(request: HttpRequest, pk: str) -> HttpResponse:
     "uploaders_tennworkcompcl",
     "uploaders_tennworkcompapp",
     "uploaders_me",
+    "uploaders_mo",
+    "uploaders_moctapp",
+    "uploaders_moctapped",
+    "uploaders_moctappsd",
+    "uploaders_moctappwd",
+    "uploaders_miss",
+    "uploaders_missctapp",
 )
 @async_to_sync
-async def court_publish_page(request: HttpRequest, pk: int) -> HttpResponse:
+async def court_publish_page(request: HttpRequest, pk: str) -> HttpResponse:
     """Display upload form and intake Opinions for partner courts
 
     :param request: A GET or POST request for the page
     :param pk: The CL Court ID for each court
     """
-    if pk not in ["tennworkcompcl", "tennworkcompapp", "me"]:
+
+    available_courts = [
+        "tennworkcompcl",
+        "tennworkcompapp",
+        "me",
+        "mo",
+        "moctapp",
+        "moctapped",
```

Comment:
if we are going to segment to moctapp ed wd and sd - we should probably not include a moctapp right?

Reply:
right, on the official site the opinions are only for each district

![image](https://github.com/freelawproject/courtlistener/assets/6474994/c2b5d15f-b52d-44fc-9130-d574dcea6fa1)

i already removed it